features/kinderen.feature

```diff
@@ -15,61 +16,188 @@ Functionaliteit: Kinderen van een persoon raadplegen
-        | 59        | Daan               | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
-      En er zijn geen gegevens geregistreerd over de identificatienummers (groep 01), naam (groep 02) of geboorte (groep 03) in categorie 9
-      Als de persoon met burgerservicenummer 555550002 wordt geraadpleegd met fields=kinderen
+      | Categorie | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
+      | 9         |                   |                     |                       |                       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+      | 59        | Daan              | de                  | Vries                 | 20031107              | 0518                      | 20031109               | PL gerelateerde               | 20031107                        | 20031109                   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550002                       |
+      | fields              | kinderen                        |
       Dan bevat het antwoord GEEN kinderen
 
+    @gba
+    Scenario: met fields vragen om een gegeven zonder waarde
+      Gegeven de persoon met burgerservicenummer 555550001 heeft de volgende kinderen in de registratie
+      | Categorie | Burgerservicenummer (01.20) | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Geboortedatum (03.10) | Gemeente document (82.10) | Datum document (82.20) | Beschrijving document (82.30) | Ingangsdatum geldigheid (85.10) | Datum van opneming (86.10) |
+      | 9         |                             |                   |                     | .                     |                       | 1926                      | 20040105               | D27894-2004-A782              | 20031107                        | 20040112                   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999996150                       |
+      | fields              | kinderen.burgerservicenummer    |
+      Dan heeft de persoon met burgerservicenummer '555550001' exact 1 kind
```

Comment:
En hoe ziet dat kind er in dit geval uit dan ? wordt er een lege property burgerservicenummer geretourneerd ? Wel relevant om dat expliciet te vermelden hier lijkt mij.

Reply:
Hier ontbreekt inderdaad een Dan regel. Het kind bevat dan geen enkel gegeven

Reply:
@MelvLee ik heb nu toegevoegd: "En het kind bevat geen enkel gegeven"
Heb jij daar in je automatisering een andere suggestie voor?

Reply:
oh, ik zie dat ik dat op een andere plek anders opgelost heb:
```
En heeft het kind alleen de volgende gegevens
      | naam | waarde  |
      | type | GbaKind |
```

Reply:
GbaKind bestaat niet meer.

in dit geval wordt het kind niet geleverd

Reply:
GbaKind bestaat niet meer.

in dit geval wordt het kind niet geleverd